### DISCUSSION TITLE
A current version for Linux

### DISCUSSION THREAD

**Discussion by fatman-del** [Purfview/whisper-standalone-win#332] [Q&A] [answered]
Hello,
the project is really wonderful. I particularly like the integration of diarize in the current version. Are there plans to make a version with diarization available for Linux as well?
thx

**Comment by Purfview** [comment, marked as answer]
Hi, currently I've no plans for new Linux release in near future.
A donation may speed things up, as there are no plans for more than one or two Linux releases in a year.
